rules/scala.bzl

```diff
@@ -535,3 +535,16 @@ configure_zinc_scala = rule(
     },
     implementation = _configure_zinc_scala_implementation,
 )
+
+def configure_zinc_scala(**kwargs):
```

Comment:
what's the reasoning behind wrapping the original configue_zinc_scala? couldn't we just use these selects on the original attributes instead? That way we don't lose the documentation

Reply:
As far as I can tell, `select` only applies in `BUILD` or macro. `select` needs to be resolved before passing to a `rule`.

Reply:
Ah it seems you're right.. 
(https://github.com/bazelbuild/bazel/issues/287#issuecomment-426735898)

That's unfortunate though cause it means we'll lose the signature docs.. an option is to explicitly propagate each argument instead of using `**kwargs`, but we'd still lose the argument types/descriptions so I'm not sure that'd be much better

Reply:
Yeah, those explicitly listed attributes will only be shown `optional`.

James said it used to work, but I really doubt it.

Reply:
It might be good to add a separate doc page for `configure_zinc_scala` and to link to it from the README. You could simply copy the old md and put it in its own file (and perhaps include a comment on how you can use `deps_used_off` and `deps_direct_off` to disable things.

Reply:
The document has been added.